features/aanhef.feature

```diff
@@ -2,216 +2,908 @@
+    | N           | in het      | Veld          | van                 | Velzen                | Geachte mevrouw In het Veld-van Velzen |
+    | N           |             | Groenen       |                     | Groenink              | Geachte mevrouw Groenen-Groenink       |
+
+  Abstract Scenario: hoofdlettergebruik in voorvoegsels bij aanduidingNaamgebruik "<naamgebruik>", geslachtsaanduiding "<geslacht>" en <voorbeeld>
+    Gegeven het systeem heeft een persoon met de volgende gegevens
+    | naam                | waarde     |
+    | burgerservicenummer | 999992934  |
+    | geslachtsaanduiding | <geslacht> |
+    En de persoon heeft de volgende naam gegevens
+    | naam                                 | waarde          |
+    | voornamen (02.10)                    | <voornamen>     |
+    | adellijke titel of predikaat (02.20) |                 |
+    | voorvoegsel (02.30)                  | <voorvoegsel>   |
+    | geslachtsnaam (02.40)                | <geslachtsnaam> |
+    | aanduidingNaamgebruik (61.10)        | <naamgebruik>   |
+    En de partner heeft de volgende naam gegevens
+    | naam                                 | waarde                  |
+    | adellijke titel of predikaat (02.20) |                         |
+    | voorvoegsel (02.30)                  | <partner voorvoegsel>   |
+    | geslachtsnaam (02.40)                | <partner geslachtsnaam> |
+    Als personen wordt gezocht met de volgende parameters
+    | naam                | waarde                          |
+    | type                | RaadpleegMetBurgerservicenummer |
+    | burgerservicenummer | 999992934                       |
+    | fields              | naam.aanhef                     |
+    Dan bevat de persoon met burgerservicenummer '999992934' de volgende 'naam' gegevens
+    | naam   | waarde   |
+    | aanhef | <aanhef> |
+
+    Voorbeelden:
+    | voorbeeld                     | geslacht | naamgebruik | voornamen | voorvoegsel | geslachtsnaam | partner voorvoegsel | partner geslachtsnaam | aanhef                                       |
+    | voorvoegsel in kleine letters | V        | E           | Jo Rene   | in het      | Zonnetje      | 't                  | Wolkje                | Geachte mevrouw In het Zonnetje              |
+    | voorvoegsel in kleine letters | O        | E           | Jo Rene   | in het      | Zonnetje      | 't                  | Wolkje                | Geachte J.R. in het Zonnetje                 |
+    | geen voornamen                | O        | E           |           | in het      | Zonnetje      | 't                  | Wolkje                | Geachte In het Zonnetje                      |
+    | voorvoegsel met hoofdletters  | V        | E           | Jo Rene   | Op Den      | Berghe        | Van Der             | Broeck                | Geachte mevrouw Op den Berghe                |
```

Comment:
Gisteren hadden we afgesproken dat voorvoegsels worden overgenomen as-is (dacht ik). Geldt dat ook samenstellingen? Zou het dus niet 'Geachte mevrouw Op Den Berghe' moeten zijn?

Reply:
ik was nog niet zover (kan niet alles tegelijk), maar is inmiddels aangepast